Recordings page + Slight modifications

PekYC opened the pull request:
Recordings page template is done. Only the base functionality is there right now. As usual, aesthetics can still be improved, namely:

- Position of the button to open the sidebar
- Responsiveness of the embedded video

Further considerations:

- Gray overlay over the content when the sidebar is open 
- Button on the bottom right to move to the next session 

That's all for the recordings page, onto the slight modifications I made:

- Added styles for links in the base-styles.css. Now all links will no longer have the blue underline by default
- Fixed an issue with the footer for the schedule page

Racheltmz commented:
Position of button to open sidebar
- Fine for computer and tablet, for mobile it appears below the video and header, personally I think it'll be better if it appears above these 2 things.

Responsiveness of the embedded video
- I'm also uncertain about how the video should look like on mobile, perhaps this is something we can discuss.

![image](https://user-images.githubusercontent.com/76416238/135763541-d93ed006-507d-4c97-a18a-e1e8a2916ded.png)
- I personally think the close button has too much space, perhaps we can remove the padding for the class, "sidenav-header text-right"

PekYC commented:
Made changes as discussed earlier today:

- Moved sidebar open button to the top of the page for smaller screens
- Removed top and bottom padding for the 'x' in the sidebar
- Added gray overlay when sidebar is open
- Added next button to move to the next recording
   - Added rec2.html to demonstrate this

https://user-images.githubusercontent.com/91200212/135813854-9aa45878-6221-4474-b29c-33d2f48dd97b.mp4
